server/index.js

```diff
@@ -73,6 +75,38 @@ app.delete('/api/reviews/:id', (req, res) => {
   });
 });
 
+//CASSANDRA API PATHS
+app.get('/api/individual_reviews/:id', (req, res) => {
+  let id = req.params.id;
+  let allReviews = [];
+  dbCass.retrieveReviews(id, (err, reviews) => {
+    if (err) {
+      res.status(404).end('An Error Occurred');
```

Comment:
I think the response text for 404 should be something in the lines of 'record was not found' rather than indicating an error. 

But this one is not a 404 case, I think you may choose 500 code to better reflect the error.